roles/aodh/tasks/main.yml

```diff
@@ -18,7 +18,8 @@
     name: "{{ item.name }}"
     user: "{{ item.user }}"
     cmd: "{{ item.cmd }}"
-    config_dirs: "{{ item.config_dirs }}"
+    args: "{{ (item.name == 'openstack-aodh-api') | ternary('--port %s' % endpoints.aodh.port.backend_api, '') }}"
```

Comment:
for ubuntu service name is aodh-api and not openstack-aodh-api. Service names are define in openstack-meta